### DISCUSSION TITLE
Connect to Dataverse for Teams

### DISCUSSION THREAD

**Discussion by therockisme** [MscrmTools/XrmToolBox#1136] [Q&A] [answered]
Dear community & developers,
I would appreciate some short hint where I could get some help regarding the use of this amazing tool in the context of a O365 and Dataverse for Teams environment (no Dynamics etc. licence).
When trying to connect to our environment I get failure messages indicating that a connection could only be made from within the teams environment and I wonder if this can be realized with this tool.
Hope one of you pros has an idea or even experience on this!
Thanks in advance,
David

**Comment by therockisme** [comment, marked as answer]
PS: Just found an issues posted last year [view](https://github.com/MscrmTools/XrmToolBox/issues/1069) stating that CDS4T is presently not open to external applications. Is this still correct?

**Comment by MscrmTools** [comment]
Yes, this is correct. Microsoft is blocking API so they cannot connect to DV4T.
Nothing we can do on our side unfortunately

**Comment by therockisme** [comment]
All right, thanks for your quick reply! It's a pitty... let's hope they will open this up a little in future...
